.circleci/config.yml

```diff
@@ -202,7 +219,7 @@ jobs:
     steps:
       - checkout
       - setup_nix
-      - import_gpg_key
+      # - import_gpg_key
```

Comment:
just checking if this is a desired change

Reply:
nope - that was for local testing :/